plugins/modules/monitoring/airbrake_deployment.py

```diff
@@ -78,44 +98,80 @@ def main():
 
     module = AnsibleModule(
         argument_spec=dict(
-            token=dict(required=True, no_log=True),
+            token=dict(required=False),
```

Comment:
```suggestion
            token=dict(required=False, no_log=True, removed_in_version='2.14'),
```
I think the `no_log=True` needs to stay there.

Also, I've just remembered that we had an easier way to deprecate parameters than having to manually call `module.deprecate()`: just add `removed_in_version='2.14'`. I would still leave the manual `module.deprecate()` in the code though, since it explicitly mentions alternatives.